cmd/symbols/internal/api/search_sqlite.go

```
@@ -59,11 +59,12 @@ func MakeSqliteSearchFunc(operations *sharedobservability.Operations, cachedData
 
 			ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 			defer cancel()
-			size, err2 := gitserverClient.GetRepoSize(ctx, args.Repo)
+			info, err2 := db.GitserverRepos().GetByName(ctx, args.Repo)
 			if err2 != nil {
 				err = errors.New("Processing symbols using the SQLite backend is taking a while. If this repository is ~1GB+, enable [Rockskip](https://docs.sourcegraph.com/code_intelligence/explanations/rockskip).")
```

Does the error message now make sense? Do we need a retry here or something (maybe the original code did block?)

---

hmm true, maybe we should just return the err?

---

The original one?